### PR TITLE
feat: event approval status for guest view

### PR DESCRIPTION
**Pull request by mthinh**
# What ? 
- Event approval status view for guest

# Screenshot 

https://github.com/lemonadesocial/lemonade-flutter/assets/28992487/02141f53-0ebc-45a3-9dc9-3747a10bb4ab

### CODE REVIEW

File: lib/core/presentation/pages/event/event_detail_page/guest_event_detail_page/sub_pages/guest_event_approval_status_page/guest_event_approval_status_page.dart [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+              ),
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class _CircularAnimation extends StatelessWidget {
```
Comment (sieuhuflit) [inline lib/core/presentation/pages/event/event_detail_page/guest_event_detail_page/sub_pages/guest_event_approval_status_page/guest_event_approval_status_page.dart:120]: Can you moved to common Widget about this animation, this reused quite a lot

File: pubspec.lock [hunk 2]
```diff
@@ -57,6 +57,14 @@ packages:
       url: "https://pub.dev"
     source: hosted
     version: "2.0.11"
+  another_flushbar:
```
[the remark this answers is not in the document]
Comment (mthinh) [reply pubspec.lock:60]: last time saw you commit this back to repo on master

[the remark this answers is not in the document]
Comment (sieuhuflit) [reply pubspec.lock:60]: So let's remove :v

[the remark this answers is not in the document]
Comment (sieuhuflit) [reply pubspec.lock:60]: Mistake commit

File: lib/core/presentation/pages/event/event_detail_page/guest_event_detail_page/sub_pages/guest_event_approval_status_page/guest_event_approval_status_page.dart (hunk 1 above)
Comment (mthinh) [reply lib/core/presentation/pages/event/event_detail_page/guest_event_detail_page/sub_pages/guest_event_approval_status_page/guest_event_approval_status_page.dart:120]: let raise to another PR, need to check which can be configurable if we make it common

**Review by sieuhuflit — APPROVED** [review]

**Pull request merged**